ArpHeader/ArpBuild.h

```diff
@@ -52,6 +52,8 @@
 
 #if __INTEL__
 
+#if __CNUC__ < 3
```

Comment:
?

Reply:
This is to use __declspec(dllimport) / __declspec(dllexport) only on gcc2.

Reply:
There is a typo, it should be GNUC and not CNUC.

Also, these declspecs are useless on gcc2 and can be removed completely. I think they were used in some older version for BeOS (when it was not using ELF format for executables), but they are not used by the current version as far as I know.

Reply:
Fixed the typo